### ISSUE TITLE
Target label type

### ISSUE THREAD

**Issue by teamclouday** [serengil/chefboost#16]
Is it true that the `Decision` column of input training dataset should be string type?
I tried to feed integer array at first and got 0 accuracy. But converting to a string array works.

**Comment by serengil** [comment]
if the target is numerical, then regression trees will be built. otherwise, id3, c4.5, cart or chaid can be built.

if you share your input data set, and the program you are running, I can understand its root cause.

**Comment by teamclouday** [comment]
Thanks for clarification! I'm using the titanic data for classification tasks. The target is either 0 or 1. Now it makes sense that I need to convert it to string values in order not to build regression trees.  
Here is the code I'm using now:
```python
# X1, Y1 is processed titanic data
X1_train, X1_test, Y1_train, Y1_test = train_test_split(X1, Y1, test_size=0.2)
config = {'algorithm': 'C4.5'}
c45_1 = chef.fit(pd.DataFrame(
    pd.np.column_stack([
        X1_train, [str(x) for x in Y1_train]
    ]),
    columns=X1.columns.to_list() + ["Decision"]
), config=config)
```
